pubspec.yaml

```diff
@@ -29,11 +29,15 @@ environment:
 # versions available, run `flutter pub outdated`.
 dependencies:
   http: ^0.13.4
+  freezed: ^2.3.2
```

Comment:
I might use this in the future. It's a way to [create models easier](https://pub.dev/packages/freezed), but it would add some complexity because of our models within models.